write docker setup at readme (#57)
ethz-asl/grid_map_geo: README.md

```
colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to grid_map_geo
```
## Docker Setup

If your operating system doesn't support ROS 2 humble, docker is a great alternative.

> First of all, you have to build project and create image by copy and paste following command to terminal

Reviewer: ```suggestion
First of all, you have to build the project and create an  image like so:
```

```bash
## Assuimg you are in the correct project directory
docker build -t gmg .
```

> To use shortcut , you may use following command;

Reviewer: ```suggestion
To use a shortcut, you may use the following command:
```

```bash
## Assuimg you are in the correct project directory
make docker_build
```

> After create image, copy and paste the following command to terminal to run image;

Reviewer: ```suggestion
After the image is created, copy and paste the following command to the terminal to run the image:
```

```bash
## Assuimg you are in the correct project directory
docker run -it --net=host --ipc=host --privileged --env="DISPLAY"  --env="QT_X11_NO_MITSHM=1" --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" --volume="${XAUTHORITY}:/root/.Xauthority"  --entrypoint /bin/bash gmg
```

> To use shortcut, you may use following commad;

Reviewer: ```suggestion
To use a shortcut, you may use following command:
```
